<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabbingDialog</class>
 <widget class="QWidget" name="TabbingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>99</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of &amp;columns:</string>
     </property>
     <property name="buddy">
      <cstring>m_spCols</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="m_spCols">
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of &amp;rows:</string>
     </property>
     <property name="buddy">
      <cstring>m_spRows</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="m_spRows">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Spacing:</string>
     </property>
     <property name="buddy">
      <cstring>m_leSpacing</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_leSpacing"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
